config/convert.go

```diff
@@ -53,6 +62,15 @@ func FileModeVal(o *os.FileMode) os.FileMode {
 	return *o
 }
 
+// FileModeCopy returns a copy of the os.FireMode
+func FileModeCopy(o *os.FileMode) *os.FileMode {
```

Comment:
This doesn't seem to be used.

Reply:
Thanks for reviewing the PR. I have removed the unused helper functions.